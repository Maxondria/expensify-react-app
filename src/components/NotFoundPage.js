import { Link } from "react-router-dom";
import React from "react";

const NotFoundPage = () => (
  <h1>
    Oops, Not Found - <Link to="/">Back To Safety</Link>
  </h1>
);

export default NotFoundPage;
