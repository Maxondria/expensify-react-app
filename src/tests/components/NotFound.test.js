import { shallow } from "enzyme";
import React from "react";
import NotFoundPage from "../../components/NotFoundPage";

it("should render NotFoundPage correctly", function() {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
