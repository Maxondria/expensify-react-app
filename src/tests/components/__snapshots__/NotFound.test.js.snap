// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<h1>
  Oops, Not Found - 
  <Link
    to="/"
  >
    Back To Safety
  </Link>
</h1>
`;
